src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,52 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  let correctTabId = selectedTabId;
+  const ids = tabs.map(tab => tab.id);
+
+  if (!ids.includes(correctTabId)) {
+    correctTabId = tabs[0].id;
+  }
+
+  const visibleTab = tabs.find(tab => tab.id === correctTabId);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                key={id}
+                data-cy="Tab"
+                className={cn({
+                  'is-active': id === correctTabId,
+                })}
+              >
+                <a
+                  href={`#${id}`}
+                  data-cy="TabLink"
+                  onClick={() => (
+                    correctTabId !== id && onTabSelected(tab)
+                  )}
```

Comment:
use handler function to handle a click

```
  const handleTabClick = (tab) => {
    if (tab.id !== selectedTabId) {
      onTabSelected(tab);
    }
  };
```

```suggestion
                  onClick={() => handleTabClick(tab)}
```